src/Components/Wizard.js

```diff
@@ -238,10 +238,14 @@ export class SavedCompanies {
     }
 
     getUserChanged() {
-        return Cookie.get(USER_CHANGED_COOKIE) === 'true';
+        return Cookie.get(USER_CHANGED_COOKIE) ? true : false;
```

Comment:
You should be able to just convert to a boolean like this:

```suggestion
        return !!Cookie.get(USER_CHANGED_COOKIE);
```

Reply:
done